BTVN.B11/main.js

```diff
@@ -0,0 +1,86 @@
+//Bai 1: Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+const newArr = (string, lengthArr) => {
+    const arr = []
+    for (let i = 0; i < lengthArr; i++) {
+      arr.push(string)
+    }
+    return arr;
+  }
+newArr('Toàn bài khó vãi chưởng :-(', 3);
+
+//Bai 2: Viết 1 hàm để đảo ngược 1 mảng, không được dùng hàm array.reverse()
+const arr = [0, 1, false, 2, undefined, '', 3, null]
+const newArr = arr.filter(function(el) {
+    return el != null | false | undefined | null;
+});
```

Comment:
```suggestion
return arr.filter(value => value);
```
Với lại hàm đâu? ở đâu viết giữa file như thế này?

nếu là `''` (empty) thì sao